CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Added
 
+- `show-config` subcommand to display currently used configuration
```

Comment:
this is added in wrong place, uder previous release. Please move it to the top, under `## [Unreleased]` section. 

A
```
### Cast
#### Added
(...)
```

should be added there, since this is a first added PR scheduled for next release :+1:

Reply:
my bad, i just searched for Cast and added it below it...

Reply:
no worries ;)